doc/website/release-notes/iceoryx-v1-0-0.md

```diff
@@ -1,4 +1,293 @@
-# iceoryx v1.0.0
+# Change Log
+
+## [vX.X.X](https://github.com/eclipse-iceoryx/iceoryx/tree/v1.0.0) (unreleased)
```

Comment:
Something went wrong here when resolving merge conflicts, right?

Reply:
Yes indeed ... I solved it